common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -32,12 +32,14 @@ $sequence--border-color: #C8C8C8;
 
 // ====================
 
-nav.sequence-nav {
+.sequence-nav {
   // TODO (cpennington): This doesn't work anymore. XModules aren't able to
   // import from external sources.
   @extend .topbar;
-  margin: -4px 0 ($baseline*1.5);
   position: relative;
+  margin: -4px 0 ($baseline*1.5);
```

Comment:
NIT: if we can, let's move this ``-4px`` to a baseline-derived value (``$baseline/5``)